scalableminds/webknossos
# Allow dataset managers to see all workflow reports of their orga

### Steps to test:
- set up voxelyticsEnabled locally, run voxelytics workflow
- promote sample user 2 to dataset manager, log in as sample user 2. should see the workflow

### Issues:
- fixes https://discuss.webknossos.org/t/access-to-voxelytics-overview-table/1786

### TODO:
 Currently, the whole administration menu is hidden for non-team-managers. @philippotto do you think it would make sense to show it to dataset managers (that are not team managers)? Otherwise, the voxelytics workflows link is not visible to them

------
